src/main/java/com/cdancy/bitbucket/rest/domain/activities/ActivitiesCommit.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.domain.activities;
+
+import com.cdancy.bitbucket.rest.domain.commit.Commit;
+import com.google.auto.value.AutoValue;
+import org.jclouds.javax.annotation.Nullable;
+import org.jclouds.json.SerializedNames;
+
+import java.util.List;
+
+@AutoValue
+public abstract class ActivitiesCommit {
+    @Nullable
+    public abstract List<Commit> commits();
+
+    public abstract long total();
+
+    @SerializedNames({"commits", "total"})
+    public static ActivitiesCommit create(List<Commit> commits, long total) {
+        return new AutoValue_ActivitiesCommit(commits, total);
```

Comment:
Same as above: for all lists that are potentially nullable lets use Utils.nullToEmpty function.